openreview/venue/process/assignment_pre_process.js

```diff
@@ -0,0 +1,36 @@
+async function process(client, edge, invitation) {
+  client.throwErrors = true
+
+  const { groups } = await client.getGroups({ id: edge.domain })
+  const domain = groups[0]
+  const venue_id = domain.id
+  const submission_name = domain.content.submission_name.value
+  const review_name = invitation.content.review_name.value
+  const reviewers_anon_name = invitation.content.reviewers_anon_name.value
+
+  if (!review_name) {
+    return
+  }
+
+  if (edge.ddate) {
+
+    const { notes } = await client.getNotes({ id: edge.head })
+    const submission = notes[0]
+    const submissionGroupId = `${venue_id}/${submission_name}${submission.number}`
```

Comment:
Using `${}` here does not cause any errors? Wouldn't it be better to do:
```suggestion
    const submissionGroupId = venue_id + '/' +submission_name + submission.number
```

Reply:
Using `+` for string concatenation is discouraged by most linters these days as it can lead to some type conversion weirdness. Like for example if submissionName and submissionNumber were both ints then that code would add the numbers together.

Reply:
> Using ${} here does not cause any errors? Wouldn't it be better to do

It doesn't because this code is in the process function and not in the content. The API skips the $ replacement on process fields.